app/src/main/java/com/vaiwork/playlistmaker/data/FavouriteTracksRepositoryImpl.kt

```diff
@@ -25,26 +25,26 @@ class FavouriteTracksRepositoryImpl(
     }
 
     override suspend fun addTrackToFavourite(track: Track) {
-        val trackEntity = trackDbConverter.map(track)
+        val trackEntity = dbConverter.map(track)
         if (trackEntity != null)
             appDatabase.favouriteTracksDao().insertTracks(listOf(trackEntity))
         else
             appDatabase.favouriteTracksDao().insertTracks(emptyList())
     }
 
     override suspend fun deleteTrackFromFavourite(track: Track) {
-        val trackEntity = trackDbConverter.map(track)
+        val trackEntity = dbConverter.map(track)
         if (trackEntity != null)
             appDatabase.favouriteTracksDao().deleteTrackEntity(trackEntity)
     }
 
     override fun selectTrackByTrackId(trackId: Int): Flow<Track?> = flow {
-        emit(trackDbConverter.map(appDatabase.favouriteTracksDao().selectTrackByTrackId(trackId)))
+        emit(dbConverter.map(appDatabase.favouriteTracksDao().selectTrackByTrackId(trackId)))
```

Comment:
🍏  Рекомендую преобразовать dbConverter.map так, чтобы она не принимала нуллабельные значения и не содержала скрытой логики по проверке пришедшего значения.
Лучше здесь сделать эту проверку:
```suggestion
        emit(
        appDatabase.favouriteTracksDao().selectTrackByTrackId(trackId)
        ?.let { dbConverter.map(it) }
        )
```